### PR TITLE
make publish tags link to the release page with populated fields

### PR DESCRIPTION
**Pull request by jakemac53**
Updates the markdown table so that "publish tag" tags are links to a release page with all the information pre-populated.

### CODE REVIEW

**Comment by jakemac53** [comment]
I set this as ready to publish just to see the link in action and make sure it works as expected, but happy to revert that and mark it as WIP if we want to wait to release.

File: pkgs/firehose/lib/src/repo.dart [hunk 1]
```diff
@@ -65,6 +66,14 @@ class Repository {
       return '${package.name}-v${package.pubspec.version}';
     }
   }
+
+  Uri calculateReleaseUri(Package package, Github github) {
+    final tag = calculateRepoTag(package);
+    final title = '${package.name} version ${package.pubspec.version}';
```
Comment (devoncarew) [inline pkgs/firehose/lib/src/repo.dart:72]: When doing this manually, I always had the title == the tag, just to save me from having to do too much manual work during a release.

I think I'm fine with that convention - there's plenty of info in the tag to help people ID releases.

If we do want to do something different, I'd suggest `package:foo v1.2.3` as it's terse but descriptive.

**Review by devoncarew — APPROVED** [review]
I have some bikeshedding on the title to use for the release.

As discussed via chat, we probably want to rename the table column of the release tag to reduce the danger of people clicking on the link before the PR's been merged.

And, no concerns w/ releasing it now (we don't need an intermediary `-wip` version).

File: pkgs/firehose/lib/src/repo.dart (hunk 1 above)
Comment (jakemac53) [reply pkgs/firehose/lib/src/repo.dart:72]: going with `package:foo v1.2.3`, I think its just slightly better than the tag

**Comment by jakemac53** [comment]
I updated the column name - although its wrapping now and kind of ugly. @devoncarew should I remove the `(post-merge)` part?

**Comment by jakemac53** [comment]
Ok, that looks better now, PTAL @devoncarew

**Review by devoncarew — APPROVED** [review]
lgtm

**Pull request merged**